larkspur/larkspur.py

```diff
@@ -154,9 +187,21 @@ def __contains__(self, key):
         res = pipe.execute()
         return all(res)
 
-    def add(self, key, skip_check=False):
+    def add(self, key: Any, skip_check: Optional[bool] = False):
```

Comment:
This arg is unused, so you can remove it

```suggestion
    def add(self, key: Any):
```